### ISSUE TITLE
Color contrast ratio is insufficient on content examples for Promo Banners (and probably Side Nav)

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/vets-design-system-documentation#151]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The text inside the promo banners examples is using `.vads-u-color--gray-light` which renders an insufficient color contrast ration on a white background.

![image](https://user-images.githubusercontent.com/55992116/67897382-a2d12f00-fb34-11e9-8959-ca74f4d421b4.png)

I also notice on the Sidebar nav component the ratio seems weak to me. It passes an axe audit in my browser, BUT the parent container has an inline style of `opacity=.5` which decreases the contrast. I think this inline style needs to be removed.

<img width="1093" alt="with-opacity-declaration" src="https://user-images.githubusercontent.com/55992116/67897610-28ed7580-fb35-11e9-9455-ee8439c53026.png">
<img width="1053" alt="without-opacity-declaration" src="https://user-images.githubusercontent.com/55992116/67897611-28ed7580-fb35-11e9-9c78-46aecaee86ce.png">

## What I expected to happen

The color contrast between text and background would be greater and meet minimum WCAG 2.0 requirements.

## Reproducing
- Formation version: current documentation site
- Device: Macbook
- Browser: Chrome

Steps to reproduce:

1. Navigate to this page https://design.va.gov/components/promo-banners
2. View the examples
3. Navigate to this page https://design.va.gov/components/sidenav
4. View the examples

**Comment by KevinMHoffmanUSDS** [comment]
Thanks, @andaleliz! So that I understand you clearly, you're pointing out a contrast issue in the design.va.gov site itself, and not one that is actually in production in live examples of these components on the site?

**Comment by andaleliz** [comment]
Yes, that's correct. I haven't seen an issue on the va.gov site itself.

**Comment by caw310** [comment]
Closing
